accepted/2023/owner-author-pmui.md

```diff
@@ -0,0 +1,78 @@
+# ***Display Trusted Owners in PM UI***
+<!-- Replace `Title` with an appropriate title for your design -->
+
+- Donnie Goodson ([donnie-msft](https://github.com/donnie-msft))
+- GitHub Issue https://github.com/NuGet/Home/issues/12501
+
+## Summary
+
+When available, highlight in the Packages Manager UI (PM UI) that the owner is trusted by the package source with a NuGet package icon. 
+In the case of `nuget.org`, also provide a hyperlink to the owner's profile page (ie, `https://www.nuget.org/profiles/<owner name>`).
+A Trusted Owner is defined as an owner that's indicated by a package source's [Search API](https://learn.microsoft.com/nuget/api/search-query-service-resource#search-result), as opposed to the field in `.nuspec`.
+
+## Motivation 
+
+<!-- Why are we doing this? What pain points does this solve? What is the expected outcome? -->
+
+Customers need a clear understanding of who created a NuGet package in order to trust that package.
+The term "owner" has become overloaded as NuGet has evolved, which has caused confusion. 
+The "author" metadata is also misleading as it's a free-form text field which doesn't imply any credentials.
+
+## Explanation
+
+### Functional explanation
+
+<!-- Explain the proposal as if it were already implemented and you're teaching it to another person. -->
+<!-- Introduce new concepts, functional designs with real life examples, and low-fidelity mockups or  pseudocode to show how this proposal would look. -->
+
+![Owners from NuGet.org are shown beside each package ID with a hyperlink and a NuGet Package Icon](../../meta/resources/OwnerAuthor/ownerNuGetOrgLinks.png)
```

Comment:
I find the many little nuget icons a little noisy. What do you think about only having the last one or exclude them entirely and just having the links? I take it you mimicked the icon on the package details link already present.

Might be worth a quick pass from the UX team

Reply:
Yeah, unfortunately UX team cancelled their office hours last week when I was going to get their input.

Absolutely it does feel noisy. I'd say that noise is good when it reiterates trust. However, now that I'm moving towards profile links (and you seem to agree with the direction), that's probably sufficient on its own.

I think customers would pick up that links are owners, and anything else is questionable/ needs more investigation.